scipy/stats/tests/test_multivariate.py

```diff
@@ -186,6 +191,30 @@ def test_degenerate_distributions(self):
                 assert_equal(pdf_rr_orth, 0.0)
                 assert_equal(logpdf_rr_orth, -np.inf)
 
+    def test_degenerate_distributions_with_inverse_cov(self):
```

Comment:
I don't think we'll need this test because I don't see why should use a precision matrix representation to work with matrices that aren't strictly positive definite. I plan to add a similar class for an eigendecomposition representation and a diagonal matrix representation, and those will be suitable for matrices that aren't positive definite.